### PR TITLE
version update 2.4.0crp and changelog

### PR DESCRIPTION
**Pull request by psychocoderHPC**
- update version.hpp
- add changelog for 2.4.0crp

PR is against the release branch `release-2.4.0crp`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,27 @@
 Change Log / Release Log for mallocMC
 ================================================================
 
+2.4.0crp
+--------
+**Date:** 2020-04-28
```
Comment (sbastrakov) [inline CHANGELOG.md:6]: Date is wrong.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,27 @@
...
+
+This release removes the boost dependency and switched to C++11.
```
Comment (sbastrakov) [inline CHANGELOG.md:8]: Perhaps capitalize `Boost` and `switches` instead of `switched`.

Comment (sbastrakov) [reply CHANGELOG.md:8]: Do you want to add the removal of Boost dependency to one of the bullet points below?

File: CHANGELOG.md (hunk 1 above)
Comment (psychocoderHPC) [reply CHANGELOG.md:6]: yes should be `May`

**Comment by psychocoderHPC** [comment]
updated

**Comment by sbastrakov** [comment]
@ax3l do you want to take a look? Or I can merge as is.

**Comment by psychocoderHPC** [comment]
@sbastrakov You can merge it

**Comment by ax3l** [comment]
@sbastrakov thanks for the ping, sorry did not get through my notifications today and we had a public holiday on Monday :)

**Pull request merged**